.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  # Dependencies listed in go.mod
+  - package-ecosystem: "gomod"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+
+  # Dependencies listed in .github/workflows/*.yml
```

Comment:
It is unclear what does it mean by `Dependencies listed in .github/workflows/*.yml`. Do you have any pointer explaining this?

Reply:
This will update actions used in workflows.